editor/editor_properties_array_dict.cpp

```diff
@@ -480,7 +485,28 @@ bool EditorPropertyArray::_is_drop_valid(const Dictionary &p_drag_data) const {
 		return true;
 	}
 
-	return false;
+	if (p_drag_data["type"] != "nodes") {
+		return false;
+	}
+
+	if (allowed_type.is_empty()) {
+		// No type requirements specified so any type is valid.
+		return true;
+	}
+
+	Array nodes = p_drag_data["nodes"];
+
+	bool is_drop_allowed = true;
+	for (int i = 0; i < nodes.size(); i++) {
+		Node *dropped_node = get_tree()->get_edited_scene_root()->get_node(nodes[i]);
```

Comment:
```suggestion
		Node *dropped_node = get_tree()->get_edited_scene_root()->get_node_or_null(nodes[i]);
```
Otherwise there would be double error in the ERR below.